src/js/view/controlbar.js

```diff
@@ -307,6 +312,12 @@ define([
         onCastActive : function(model, val) {
             utils.toggleClass(this.elements.cast.element(), 'jw-off', !val);
         },
+        onAirplayAvailable : function(model, val) {
+            this.elements.airplay.toggle(val);
+        },
+        onAirplayActive : function(model, val) {
+            utils.toggleClass(this.elements.airplay.element(), 'jw-off', !val);
+        },
```

Comment:
I would need to extend `controlbar.js` into commercial similarly to how we do it for `view.js`, which we don't have right now. Should I make that change, or leave it as tech debt?